app/assets/javascripts/incident_reports/edit.coffee

```diff
@@ -17,7 +17,13 @@ determineShouldProvideReasonNotUpToCurb = ->
 
 addFields = (fieldsSelector) ->
   fields = $(fieldsSelector).last()
-  fields.clone().insertAfter fields
+  field = fields.clone()
+  for attrs in field.find('input')
```

Comment:
I think I'd prefer a method that takes some string containing a number and returns the same string with the number incremented.

Also, it's not a complete mystery where in the string the number will fall, right? Could we write a more specific (and thereby more descriptive) regex?

Reply:
Ok, I'll try

Reply:
@dfaulken why do you want the method to take that string and return the string with the number incremented? The reason I did this was to show that only the number was getting changed.

Reply:
Right. Agreed! I just think that the current code is a little cluttered with the combination of the `attrs` logic *and* the number-incrementing logic. Whereas if we abstract one into a function, we get clearer separation of concerns.

Reply:
Not a blocking issue - if you think that that's a step backwards in readability I'm happy to approve as is. It's only a few lines.

Reply:
OHHH. A method. Got it.